glide.yaml

```diff
@@ -26,9 +27,11 @@ import:
   repo: https://github.com/quantum/goautoneg.git
 - package: github.com/coreos/pkg/capnslog
 - package: k8s.io/client-go
-  version: e121606b0d09b2e1c467183ee46217fa85a6b672
+  version: release-3.0
   subpackages:
   - kubernetes/typed/core/v1
+- package: k8s.io/apimachinery
+  version: release-1.6
```

Comment:
I dont think this dictates which Kubernetes version this would work. This is to set the version of the the APIs and bindings that Rook uses to talk to Kubernetes. I havent tested with coreos-kubernetes. I tested with 1.6 Minikube. But i dont see why it wouldnt work with 1.5 coreos-kubenetes.

The main driver for updating these dependencies is to use the external-volume plugin lib. Not so much for Rook to run on kubernetes 1.6.